internal/analytics/deferred.go

```diff
@@ -18,10 +26,53 @@ type deferredData struct {
 }
 
 const deferredCfgKey = "deferrer_analytics"
+const deferrerFileName = "deferrer"
+
+func deferrerTimeStamp(cfg Configurable) string {
+	return filepath.Join(cfg.ConfigPath(), deferrerFileName)
+}
```

Comment:
This function is returning a file path, not a time stamp. Could it be modified to stat the file and return that timestamp? That or just change the name of the function.

Reply:
Good point.  Changing the names.